tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -713,6 +715,41 @@ function setupStorybook(tree: Tree, options: NormalizedSchema) {
   return tree;
 }
 
+function moveStories(tree: Tree, options: NormalizedSchema) {
+  const componentName = options.normalizedPkgName.split('-')[1];
+  const oldStoriesPath = `${options.projectConfig.sourceRoot}/stories`;
```

Comment:
**chore:** we need to use `path.join` or `joinPathFragments` to make sure this works on non "nix" OS